modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/SLAResultUtil.java

```diff
@@ -97,6 +97,9 @@ else if (Objects.equals(
 
 			return SLAResult.Status.PAUSED;
 		}
+		else if (Objects.equals(status, WorkflowMetricsSLAStatus.NEW.name())) {
```

Comment:
@seltong move this if statement to second position, please. We must use the alphabetical order here as well. :)